### DISCUSSION TITLE
How to render the lottie animation in a CGContext?

### DISCUSSION THREAD

**Discussion by luckysmg** [airbnb/lottie-ios#1790] [Q&A]
As desc.

**Comment by calda** [comment]
What are you trying to do? Can you share any sample code?

**Reply by luckysmg** [reply]
I m trying to render it in CGContext.  Then I will use this CGContext to do some other things. Is there any API I could use😄?

**Reply by calda** [reply]
You can use `CALayer.render(in: context)` to render a `UIView`'s `CALayer` in a `CGContext`. If you're using the Core Animation rendering engine, you need to use `view.layer.presentation()` instead of `view.layer`: https://github.com/airbnb/lottie-ios/discussions/1769#discussioncomment-3860672
